.circleci/config.yml

```diff
@@ -575,6 +574,17 @@ jobs:
       - run: pyenv rehash
       - run: make lint-codestyle
 
+  localization:
+    <<: *defaults
+    steps:
+      - setup_container:
+          wait_services: false
+          install_node_dependencies: true
+          install_python_dev_dependencies: true
+      - run : make extract_locales
+      - run: git diff
```

Comment:
We probably want a follow-up to make that useful - at the moment, because of differences in how pontoon formats the po files, it's a mess to read most of the time.

Reply:
Agreed.

Reply:
Baby steps.